rest_framework/authtoken/management/commands/drf_create_token.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from rest_framework.authtoken.models import Token
+
+
+class Command(BaseCommand):
+    help = 'Create DRF Token for a given user'
+
+    def create_user_token(self, username):
+        user = User.objects.get(username=username)
```

Comment:
We should probably use `get_user_model()` here. I'm also thinking this should make less assumptions regarding querying by `username`. Perhaps we can leverage the user model's `USERNAME_FIELD` together with `get_by_natural_key()`.  Similar to how [`django.contrib.auth.backends.ModelBackend`](https://github.com/django/django/blob/master/django/contrib/auth/backends.py#L12) does it.